front/cypress/e2e/report_builder/broken_report.cy.ts

```diff
@@ -0,0 +1,168 @@
+            return fieldConfig.items.oneOf.map((x: any) => x.const);
+          }
+
+          if (fieldConfig.enum) {
+            return fieldConfig.enum;
+          }
+
+          return undefined;
+        };
+
+        const selectAnswerKeys = getAnswerKeys(selectKey);
+        const multiSelectAnswerKeys = getAnswerKeys(multiSelectKey);
+
+        const firstName = randomString();
+        const lastName = randomString();
+        const email = randomEmail();
+        const password = randomString();
+        const gender = 'female';
+
+        cy.apiSignup(firstName, lastName, email, password)
+          .then((response) => {
+            userId = response.body.data.id;
+
+            cy.apiUpdateUserCustomFields(email, password, { gender });
+          })
+          .then(() => {
+            cy.apiCreateSurveyResponse(email, password, projectId, {
+              [selectKey]: selectAnswerKeys[0],
+              [multiSelectKey]: [
+                multiSelectAnswerKeys[0],
+                multiSelectAnswerKeys[1],
+              ],
+            });
+          });
+      })
+      .then(() => {
+        return cy.apiCreatePhase({
+          projectId,
+          title: randomString(),
+          startAt: moment().subtract(29, 'day').format('DD/MM/YYYY'),
+          participationMethod: 'information',
+        });
+      })
+      .then((phase) => {
+        reportPhaseId = phase.body.data.id;
+      });
+  });
+
+  after(() => {
+    cy.apiRemoveProject(projectId);
+    cy.apiRemoveUser(userId);
+  });
+
+  it('still works after deleting survey phase', () => {
+    cy.setAdminLoginCookie();
+    cy.apiCreateReportBuilder(reportPhaseId).then((report) => {
+      const reportId = report.body.data.id;
+
+      cy.visit(`/admin/reporting/report-builder/${reportId}/editor`);
+
+      cy.get('#e2e-draggable-survey-question-result-widget').dragAndDrop(
+        '#e2e-content-builder-frame',
+        {
+          position: 'inside',
+        }
+      );
+
+      cy.wait(1000);
+
+      // Select project, phase and question
+      cy.get('#e2e-report-builder-project-filter-box select').select(projectId);
+      cy.get('#e2e-phase-filter').select(surveyPhaseId);
+      cy.get('.e2e-question-select select').first().select(surveyFields[1].id);
+
+      // Save
+      cy.intercept('PATCH', `/web_api/v1/reports/${reportId}`).as(
+        'saveReportLayout'
+      );
+      cy.get('#e2e-content-builder-topbar-save').click();
+      cy.wait('@saveReportLayout');
+
+      // Go to report tab, make sure this looks good
+      cy.visit(`/admin/projects/${projectId}/phases/${reportPhaseId}/report`);
+      cy.get('.e2e-survey-question-ungrouped-bars');
+
+      // Now delete the survey phase
+      cy.intercept('DELETE', `/web_api/v1/phases/${surveyPhaseId}`).as(
+        'removePhase'
+      );
+      cy.apiRemovePhase(surveyPhaseId);
+      // This phase deleting is insanely slow and for some reason the intercept doesn't work
```

Comment:
May it be because the `intercept` URL starts with `/` and the URL in `apiRemovePhase` doesn't? Or maybe we need `**` as in [some other places](https://github.com/CitizenLabDotCo/citizenlab/blob/36d5ddcd6029a3b05aabd42e72f2e7c5dfb3e326/front/cypress/e2e/impact_tracking.cy.ts#L30)?

If none of it, do we still need to intercept this request if we don't wait for it (there's no `cy.wait('@removePhase')`?